common/gratia/common/condor_ce.py

```diff
@@ -362,9 +367,15 @@ def queryJob(jobid):
     certinfo = {}
     if 'x509UserProxyVOName' in info:
         certinfo["VO"] = info['x509UserProxyVOName']
+    elif 'AuthTokenIssuer' in info:
+        certinfo["VO"] = info['AuthTokenIssuer']
     if 'x509userproxysubject' in info:
         certinfo['DN'] = info['x509userproxysubject']
+    elif 'AuthTokenSubject' in info:
+        certinfo['DN'] = info['AuthTokenSubject']
     if 'x509UserProxyFirstFQAN' in info:
         certinfo['FQAN'] = info['x509UserProxyFirstFQAN']
+    elif 'AuthTokenIssuer' in info:
+        certinfo["FQAN"] = info['AuthTokenIssuer']
```

Comment:
I'm not sure this actually makes sense. @djw8605 does the GRACC depend on the certinfo FQAN, which currently corresponds to the first VOMS attr of the proxy

Reply:
Is this used in determining the VOName or ReportableVOName?

Reply:
So as it stands here, the AuthToken attrs are used as follows -

For certinfo:
- AuthTokenIssuer -> VO & FQAN
- AuthTokenSubject -> DN

For gratia usage records:
- AuthTokenIssuer -> VOName & ReportableVOName
- AuthTokenSubject -> DN

I fuzzily remember that this had been the result of some conversation, but I am having trouble tracking down where the decision about what to do for FQAN happened (if it actually did).  I'm open to input in any case.

Reply:
@edquist what is the certinfo FQAN used for?

Reply:
Ah, @brianhlin, always with the good questions!  :)

https://github.com/opensciencegrid/gratia-probe/blob/v2.0.1-2/common/gratia/common/certinfo.py#L105-L110

So, if there is no DN in the certinfo, then FQAN gets used for VOName (and VO from certinfo gets used for ReportableVOName)

Although, since we put AuthTokenSubject into the certinfo DN, that above path is not taken.  A little bit further down you see the same result though (that is, if DN _is_ in certinfo)

https://github.com/opensciencegrid/gratia-probe/blob/v2.0.1-2/common/gratia/common/certinfo.py#L140

I guess that means in all cases AuthTokenIssuer maps to both VOName & ReportableVOName.

Reply:
Ok, in the interest of dropping unnecessary code paths in the future, let's not bother setting the certinfo FQAN:

```suggestion
```

Feel free to add a comment indicating this decision so that we're not confused by the asymmetry in the future